codecparsers/mpeg2_parser.cpp

```diff
@@ -255,8 +255,8 @@ namespace MPEG2 {
             return false;
 
 
-        DEBUG("slice header size                  : %ld",
-              m_slice.sliceHeaderSize);
+        DEBUG("slice header size                  : %lld",
```

Comment:
use [PRIu64 ](http://en.cppreference.com/w/cpp/types/integer)in string maybe better